platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -966,8 +966,10 @@ public void run() {
             mapboxMap.onPostMapReady();
           }
         });
+      } else if (change == DID_FINISH_RENDERING_FRAME || change == DID_FINISH_RENDERING_FRAME_FULLY_RENDERED) {
```

Comment:
this is high traffic code, every small performance gain we can get, we should leverage